app/src/org/commcare/preferences/CommCareServerPreferences.java

```diff
@@ -45,32 +48,43 @@
         prefKeyToAnalyticsEvent.put(PREFS_SUPPORT_ADDRESS_KEY, GoogleAnalyticsFields.LABEL_SUPPORT_EMAIL);
     }
 
+    @NonNull
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
+    protected String getTitle() {
+        return Localization.get("settings.server.title");
+    }
 
-        PreferenceManager prefMgr = getPreferenceManager();
-        prefMgr.setSharedPreferencesName(CommCareApplication.instance().getCurrentApp().getPreferencesFilename());
-        addPreferencesFromResource(R.xml.server_preferences);
+    @Nullable
+    @Override
+    protected Map<String, String> getPrefKeyAnalyticsEventMap() {
+        return prefKeyToAnalyticsEvent;
+    }
 
-        GoogleAnalyticsUtils.reportPrefActivityEntry(GoogleAnalyticsFields.CATEGORY_SERVER_PREFS);
+    @Nullable
+    @Override
+    protected Map<String, String> getPrefKeyTitleMap() {
+        return null;
+    }
 
-        setTitle(Localization.get("settings.server.title"));
-        CommCarePreferences.addBackButtonToActionBar(this);
+    @NonNull
+    @Override
+    protected int getPreferencesResource() {
+        return R.xml.server_preferences;
+    }
 
-        GoogleAnalyticsUtils.createPreferenceOnClickListeners(prefMgr, prefKeyToAnalyticsEvent,
-                GoogleAnalyticsFields.CATEGORY_SERVER_PREFS);
+    @Override
+    public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
+        // No listeners
```

Comment:
Can you add this same comment to the other empty implementations of this method? I think it's useful to make it clear that it's intentional